src/components/common/SectionSubtitle.vue

```diff
@@ -0,0 +1,55 @@
+<script>
+import { defineComponent } from 'vue';
+export default defineComponent({
+  props: {
+    size: {
+      type: String,
+      default: 'medium',
+      validator(value) {
+        return [
+          'large',
+          'medium'
+        ].includes(value);
+      },
+    },
+  }
+});
+</script>
+
+<template>
+  <p class="section-subtitle" :size=this.size>
```

Comment:
Consider always using quotes `""` when binding an attribute. Even though it works, it breaks syntax highlighting on GitHub.

Also, you don't need to reference props via `this` keyword inside `<template>`: Just use `:size="size"`.

Reply:
Wow, I was using "this" probably in every template so far 😄 Good to know, thanks a lot! I will correct the quotes as well.